js/adopters_test.js

```diff
@@ -0,0 +1,83 @@
+const eclipsefdn_adopters_element = document.querySelectorAll(".eclipsefdn-adopters")[0];
+
+function testAdopters(working_group) {
+	fetch("/adopters/assets/js/adopters.json")
```

Comment:
Additionally, I don't know if this URL would work on Netlify as things are mounted at the root level rather than on `/adopters`. We could pass this in as a variable instead, and just generate it via Hugo when we build the page and script tag

Reply:
change to use baseURL